pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -228,30 +236,34 @@ def on_epoch_end(self, epoch, logs=None):
         self.epochs_since_last_save += 1
         if self.epochs_since_last_save >= self.period:
             self.epochs_since_last_save = 0
-            filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
-            if self.save_best_only:
+            if self.save_all or self.save_last:
+                overwrite = not self.save_all
+                filepath = '{}/{}_ckpt_epoch_{}.ckpt'.format(self.filepath, self.prefix, epoch + 1)
```

Comment:
what is the difference between `all` and `last` if they have the same name co last contains epoch name and the previous is not removed...

Reply:
the previous *is* removed  if last is True and all is False ('overwrite=not save_all')

Reply:
@ceyzaguirre4 but the name is different (unique by the epoch id) right?